test/e2e/common/secrets.go

```diff
@@ -180,14 +201,19 @@ func secretForTest(namespace, name string) *api.Secret {
 	}
 }
 
-func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32) {
+func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32, secret2 *api.Secret) {
```

Comment:
I want to be explicitly clear that this test is about two secrets and that is why I chose this approach.
Just passing a secretName does not make this apparent.
Sure I can add a comment and state this but I'd rather the code speak for itself that is why I stuck with 'secret2'.